### DISCUSSION TITLE
New builds never try to connect to wifi.

### DISCUSSION THREAD

**Discussion by rledonne** [forkineye/ESPixelStick#644] [General]
I have downloaded a few different of the newest builds and cannot get any of them to connect (or look like they are even trying) to connect to wifi.
The v4 beta 4 will install and connect, but if I use any of the CI builds, it doesn't seem to do anything after it finishes flashing.  
The device is a dig-quad (not AE+).  I have tried with two different devices, and they both do the same thing.

The serial output after flashing just hangs here.

ets Jul 29 2019 12:21:46
16:27:50: 
16:27:50: rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
16:27:50: configsip: 0, SPIWP:0xee
16:27:50: clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
16:27:50: mode:DIO, clock div:2
16:27:50: load:0x3fff0030,len:1184
16:27:50: load:0x40078000,len:13192
16:27:50: load:0x40080400,len:3028
16:27:50: entry 0x400805e4
16:27:50: E (498) esp_core_dump_flash: No core d�f�� partition found!
16:27:50: E (499) esp_core_dump_flash: No core dump partition found!
16:27:50: 
16:27:50: [    ESP] ESPixelStick v4.0-ci5659517986 (Jul 25 2023 - 16:50:14)
16:27:50: [    ESP] v4.4.4
16:27:50: [FileMgr] Flash file system initialized. Used = 204800 out of 327680
16:27:51:

**Comment by MartinMueller2003** [comment]
This is a duplicate of https://github.com/forkineye/ESPixelStick/issues/643

**Reply by rledonne** [reply]
I am not sure this is a duplicate of #643 after reading through it.  Maybe it is related, but I never get a connection at all, so I can't view the UI hanging.

![image](https://github.com/forkineye/ESPixelStick/assets/24280662/403025f2-417e-4408-8cbb-77d237982bd2)

I would expect it to show some sort of wifi information after the last line of the serial output, but it hangs there.

**Reply by MartinMueller2003** [reply]
Both have the same root problem. There have been updates to the base OS code and IDF that are causing issues with some of the versions of support libraries we are using. I found the magic combination for the ESP8266 and now I am trying to find the right combination to get the ESP32 builds to work again.

**Comment by patdelaney** [comment]
Is the 8266 version available to test?

**Comment by MartinMueller2003** [comment]
Only in my repo. I am creating a new dist file now. It takes an hour or two.

**Comment by MartinMueller2003** [comment]
I am including a fix for the ESP32 as well. The problem was a change in the SPI driver that resulted in the File Manager to spend forever trying to reset the SPI bus and failing to do so. This is now fixed.

**Comment by MartinMueller2003** [comment]
New dist.zip is at https://drive.google.com/drive/folders/1bGrkXSMoGjWtX5iWspPFRogM2G6c1VXu?usp=sharing

**Comment by MartinMueller2003** [comment]
Fixed in PR 645
